src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -29,28 +34,19 @@ public override SarifLogVersionOne VisitSarifLogVersionOne(SarifLogVersionOne v1
             return null;
         }
 
-        public static ExceptionData CreateExceptionData(ExceptionDataVersionOne v1ExceptionData)
+       internal ExceptionData CreateExceptionData(ExceptionDataVersionOne v1ExceptionData)
         {
             ExceptionData exceptionData = null;
 
             if (v1ExceptionData != null)
             {
                 exceptionData = new ExceptionData
                 {
+                    InnerExceptions = v1ExceptionData.InnerExceptions?.Select(e => CreateExceptionData(e)).ToList(),
```

Comment:
Hah! You're becoming the LINQ whisperer. Pro tip: If the right-hand side of a lambda is a single call to a function whose signature matches the lambda's signature, you can just say its name:
```
InnerExceptions = v1ExceptionData.InnerExceptions?.Select(CreateExceptionData).ToList()
```

Reply:
Ah! cool

Reply:
Fixed